readthedocs/vcs_support/backends/git.py

```diff
@@ -214,19 +214,28 @@ def lsremote(self):
         self.check_working_dir()
         code, stdout, stderr = self.run(*cmd)
 
-        tags = []
         branches = []
+        all_tags = {}
+        light_tags = {}
```

Comment:
What's the meaning of `light_tags`?

Reply:
git has annotated and lightweight tags, lightweight tags are the "normal" ones.

Reply:
👍🏼 --would be good to explain this in a code comment.